subprojects/snapshots/src/main/java/org/gradle/internal/snapshot/FileSystemSnapshotVisitor.java

```diff
@@ -31,7 +31,7 @@
     /**
      * Called for each regular/missing file.
      */
-    void visit(FileSystemLocationSnapshot fileSnapshot);
+    void visitFile(FileSystemLocationSnapshot fileSnapshot);
```

Comment:
I think the Javadoc needs to be updated to include unavailable files.

Reply:
Done.

Reply:
And I guess now it has to be reverted? :o

Reply:
Well, it still will receive the unavailable files, the file type of those will be missing, though...

I am fine either way.

Reply:
Me too. Let's leave it as it is.